chocolatey/choco
# (maint/doc) Remove special case for List page title in GenerateDocs

## Description Of Changes

- Removed special case in generatedocs which made the List page title List/Seach instead

## Motivation and Context

We have individual pages for all our command aliases, so we don't need this anymore, and it's also now wrong since List is not the same command as Search anymore.

## Testing

1. Build
2. Run GenerateDocs.ps1
3. Ensure the `docs/generated/choco/commands/list.md` file does not have its page title set to `List/Search` and is just `List`

### Operating Systems Testing

N/A

## Change Types Made



## Change Checklist

    - This will be done during the v2 release when we regenerate command documentation proper.

## Related Issue


N/A, build / docgen change.
